trappy/ftrace.py

```diff
@@ -306,14 +367,17 @@ def __populate_data(self, fin, cls_for_unique_word):
 
             if not self.basetime:
                 self.basetime = timestamp
+                # Disregard windows, this way the cache will be ready for later
+                # window requests
+                self.max_window = self._max_window()
+                self.window = (0, None)
+                self.abs_window = (0, None)
```

Comment:
Instead of setting a window that does nothing here, why don't we just delete the __before/after_window checks (and methods themselves) below? 

AFAICT `self.window` and `self.abs_window` are no longer used except to calculate `self.max_window`.

Reply:
If we want to truly commit to having cache enabled by default, I suppose we could, yes.